README.md

```diff
@@ -392,6 +373,41 @@ for movement in account.movements.all():
     print(movement.id)
 ```
 
+### Direct Debit managers
```

Comment:
Siento que está raro el nivel de esto, está al mismo nivel que `The shape of the SDK` y `Serialization`, quizás debería estar con 4 asteriscos y los subtítulos ser casi que un texto en bold nomás

Reply:
Si la verdad no sabía donde poner esto. Quizás no tiene tanto sentido y mejor dejar los managers solamente.

Reply:
Quizás es buena idea dejar los managers nomás y que entiendan el producto en la documentación misma